chainregistry.go

```diff
@@ -33,14 +33,14 @@ import (
 )
 
 const (
-	defaultBitcoinMinHTLCMSat   = lnwire.MilliSatoshi(1000)
-	defaultBitcoinBaseFeeMSat   = lnwire.MilliSatoshi(1000)
-	defaultBitcoinFeeRate       = lnwire.MilliSatoshi(1)
+	defaultBitcoinMinHTLCMSat   = lnwire.MilliSatoshi(0)
```

Comment:
This is okay for now, but can we add an issue to track this? We might want to change it back at some point (so as to not affect LND)